benchmarks/src/main/scala/bloop/logging/NoOpLogger.scala

```diff
@@ -0,0 +1,13 @@
+package bloop.logging
+
+object NoOpLogger extends AbstractLogger {
```

Comment:
I think `Noop` is an standard name :smile:

Reply:
Good point. I hope APFS has case sensitive file names...